pkg/satellite/satelliteweb/satelliteql/mutation.go

```diff
@@ -51,6 +53,30 @@ func rootMutation(service *satellite.Service, types Types) *graphql.Object {
 					return user.ID.String(), nil
 				},
 			},
+			deleteUserMutation: &graphql.Field{
+				Type: types.User(),
+				Args: graphql.FieldConfigArgument{
+					fieldID: &graphql.ArgumentConfig{
+						Type: graphql.NewNonNull(graphql.String),
+					},
+				},
+				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+					id, _ := p.Args[fieldID].(string)
+
+					idBytes, err := uuid.Parse(id)
+					if err != nil {
+						return nil, err
+					}
+
+					user, err := service.GetUser(p.Context, *idBytes)
```

Comment:
Do we need to return the user that was deleted? I'm not sure, whether it's necessary.